### PR TITLE
[WIP] Remove Hard Reload From PDP After Publishing changes

### PR DESCRIPTION
**Pull request by michael-alade**
#**Resolves**
#2246 

**Testing**
1. Login as admin
1. Create a new product.
1. Set variant label to something
1. Click the edit button next to the social icons
1. Open the product settings accordion
1. Set the product title (and the perma-link will be set as well)
1. Hit publish
1. Observe that the PDP does not reload.

### CODE REVIEW

File: imports/plugins/included/product-detail-simple/client/containers/publishContainer.js [hunk 1]
```diff
@@ -37,27 +36,11 @@ class ProductPublishContainer extends Component {
     }
   }
 
-  handlePublishSuccess = (result) => {
-    if (result && result.status === "success" && this.props.product) {
-      const productDocument = result.previousDocuments.find((product) => this.props.product._id === product._id);
-
-      if (productDocument && this.props.product.handle !== productDocument.handle) {
-        const newProductPath = Router.pathFor("product", {
```
Comment (kieckhafer) [inline imports/plugins/included/product-detail-simple/client/containers/publishContainer.js]: This should be Router.go

File: imports/plugins/included/product-detail-simple/client/containers/publishContainer.js [hunk 2]
```diff
@@ -37,27 +36,11 @@ class ProductPublishContainer extends Component {
...
-          hash: {
-            handle: this.props.product.handle
-          }
-        });
-
-        window.location.href = newProductPath;
```
Comment (kieckhafer) [inline imports/plugins/included/product-detail-simple/client/containers/publishContainer.js]: Remove this

**Comment by mikemurray** [comment]
To mirror what @kieckhafer mentioned. Everything that was deleted should be undeleted. Change the `Router.pathFor` to a `Router.go`. remove the `window.location.href`. It was always the intention to forward the browser to the new URL. the `window.location.href` was a hack to get around a flow router issue.

This probably won't solve the issue of the grid reactivity issue.

**Comment by michael-alade** [comment]
@mikemurray I tried the `Router.go` but I'm getting this error.
screen shot 2017-06-06 at 9 21 24 am

**Comment by zenweasel** [comment]
Closing as we seem to have reached an impasse. Notes should be made on the ticket in regards to approaches tried, etc.

**Comment by michael-alade** [comment]
Having tried to use the `Route.go` for redirect the seller to the new product permalink after publishing the product, I still got another issue with the product not being found. Also, for the product grid, I tried using subscription manager but it seems to be the wrong approach. We think this is caused by the Revisions collection being updated when publishing the product.

**Comment by zenweasel** [comment]
@michael-alade Let's make the notes on the ticket, not the PR

**Pull request closed without merging**